### DISCUSSION TITLE
feature requests

### DISCUSSION THREAD

**Discussion by JIGASHI** [FreeTubeApp/FreeTube#4344] [General]
Could you guys make an editor inside the seek bar where you can mark A/B interval and a construct a native downloader mp3/m4a and mp4 if video. So you can download just a part of the video/audio so you don't have to grab link, download the entire thing and do the whole editing thing, delete.

What about adding support for rumble and other sites like odysee, bitchute which you can switch between like profiles.

**Comment by kommunarr** [comment]
1. That is an interesting feature idea, but a very low priority from an implementation time to utility ratio compared to our other current requests.
2. [See our answer here.](https://docs.freetubeapp.io/faq/#will-you-consider-adding-support-for-services-outside-of-youtube-twitch-soundcloud-peertube-etc)

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
1. There is an issue open for this https://github.com/FreeTubeApp/FreeTube/issues/903

**Comment by skyrl69** [comment]
That would be neat to be able to configure which formats have to be shown in the drop down list under download button
That would be grand to allow download/convert in mp3 format (even through mp3lame encoder)
